sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -295,6 +295,94 @@ private void AddDiagnostic(DiagnosticLevel level, string message, Cursor cursor)
             _diagnostics.Add(diagnostic);
         }
 
+        private void AddCppAttributes(ParmVarDecl parmVarDecl, string prefix = null, string postfix = null)
```

Comment:
Just want to confirm. This basically saves all the C++ attributes separated by `^` in a C# style attribute?